tests/haskell.vader

```diff
@@ -0,0 +1,39 @@
+Include: _setup.vader
```

Comment:
I'd prefer it to be in `tests/haskell.vader` - might need Makefile adjustments etc though, so I can do it.